.github/workflows/pull-request.yml

```
@@ -6,7 +6,7 @@ on:
 
 jobs:
   tests:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

I'm personally not a fan of "latest" in any version and like more an exact number, to make builds always reproducible. Let's discuss this in standup, maybe with Marco

---

I was writing the same thing. You've just left the review sooner 😅 
Let's use
```suggestion
    runs-on: ubuntu-24.04
```
as we don't know if it will work on the next published image. This will save us time debugging.

---

Indeed! We can keep it as 24.04 then instead of using `-latest`